### PR TITLE
Feature/landing page update

### PR DESCRIPTION
**Pull request by AssafMiron**
## Summary by CodeRabbit

* **New Features**
  * Redesigned the landing page with interactive demos, expanded FAQs, improved navigation, and mobile download fallbacks.
  * Added dedicated FAQ and privacy pages, enhanced comparison and feature content, and a redirect from the legacy comparison link.
  * Improved accessibility, download tracking, SEO metadata, structured data, and mobile navigation.

* **Documentation**
  * Added landing-page product, UX, frontend, and QA planning documents.
  * Expanded Silent Learner and Google Antigravity/Gemini CLI guidance.
  * Corrected branding and source-build instructions.

### CODE REVIEW

File: docs/features/landing-page-update/qa-plan.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+   - Verify interactive demo tab switching and video modal opening/closing.
+   - Test mobile fallback card display on viewport widths < 768px.
+   - Validate JSON-LD syntax for `SoftwareApplication` and `FAQPage`.
+   - Verify page navigation links to `faq.html`, `vs-chatgpt.html`, `privacy.html`, `features.html`, `compare.html`, and `data-ownership.html`.
+2. **Release Plan**:
+   - Commit all changes with conventional commit message (`feat(landing): optimize GEO/AIO, hero section conversion, and mobile fallback`).
+   - Push to branch `feature/landing-page-update`.
+
+## Open Risks
+- None.
```
[the remark this answers is not in the document]
Comment (AssafMiron) [reply docs/features/landing-page-update/qa-plan.md]: Ignore

**Review by aviatam — APPROVED** [review]
Verified the follow-up fixes on the latest branch. CodeRabbit and the Ubuntu test suite are green, and I also ran local backend tests, build, the targeted Silent Learner E2E, and diff whitespace checks.

**Review by aviatam — APPROVED** [review]
